scripts/ci/jobs/diff-dumps.sh

```diff
@@ -121,13 +121,19 @@ upload_offline_dump() {
     latest_version=$(grep -oE '^[0-9]+\.[0-9]+' "$version_file" | sort -V | tail -n 1)
 
     file_to_check="scanner-v4-defs-${latest_version}.zip"
+    temp_file="/tmp/$file_to_check"
 
-    if curl --silent --show-error --max-time 60 --retry 3 -o $file_to_check https://definitions.stackrox.io/v4/offline-bundles/$file_to_check; then
-        # If the file exists, add it to scanner-vuln-updates.zip
-        zip scanner-vuln-updates.zip "$file_to_check"
-        echo "$file_to_check added to scanner-vuln-updates.zip"
+    if curl --silent --show-error --max-time 60 --retry 3 -o "$temp_file" "https://definitions.stackrox.io/v4/offline-bundles/$file_to_check"; then
```

Comment:
We have `-o e` enabled; why not just let `curl` fail on its own instead of wrapping these calls in `if-else`?

Reply:
I added `jq .` to the if check so it should be ok now.